src/modules/StudentsTable/components/Dashboard/styled.ts

```diff
@@ -0,0 +1,16 @@
+import styled from 'styled-components';
+import { WHITE_COLOR } from 'appConstants/colors';
+
+export const StyledDashboard = styled.table`
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+  width: 91.66666666666667%;
```

Comment:
is it possible to round the width value or write it in more obvious units?

Reply:
It's depends. If we want the table to fully match the figma layout ("pixel perfect") then we need these values. If this is not essential, we can round this value.
Which way is the right?

Reply:
we do not need pixel perfect there. Round.